osu.Game.Rulesets.Osu/Mods/InputBlockingMod.cs

```diff
@@ -110,5 +126,14 @@ public void OnReleased(KeyBindingReleaseEvent<OsuAction> e)
             {
             }
         }
+
+        protected class NotesPerSecondSetting : BindableInt
```

Comment:
Why is this its own class?

Reply:
It's just to avoid code repetition between the two options.